classes/submission/reviewAssignment/ReviewAssignment.php

```diff
@@ -256,23 +255,23 @@ public function setRecommendation($recommendation)
     }
 
     /**
-     * Get unconsidered state.
+     * Get considered state.
      *
      * @return int
      */
-    public function getUnconsidered()
+    public function getConsidered()
```

Comment:
This (and the associated clean-up of constants) will make this so much clearer, thanks!